packages/runtime/runtime-utils/src/utils.ts

```diff
@@ -37,20 +37,25 @@ export async function seqFromTree(
 /**
  * Encode compact ID (returned by IContainerRuntime.generateDocumentUniqueId()) to a compact string representation.
  * While this is the main usage pattern, it works with any non-negative integer or a string.
- * Strings are retured as is, and assumed to be UUIDs, i.e. unique enough to never overlap with
+ * Strings are returned as is, and assumed to be UUIDs, i.e. unique enough to never overlap with
  * numbers encoded as strings by this function. Any other strings are likely to run into collisions and should not be used!
  * This function is useful in places where we serialize resulting ID as string and use them as strings, thus we are not
  * gaining any efficiency from having a number type.
- * We do not provide a decode function, so this API is only useful only result is stored and there is no need to go back to origianl form.
+ * We do not provide a decode function, so this API is only useful only result is stored and there is no need to go back to original form.
  * @param idArg - input - either a non-negative integer or a string. Strings are returned as is, while numbers are encoded in compat form
- * @param prefix - optinal string prefix
+ * @param prefix - optional string prefix
  * @returns A string - representation of an input
  * @internal
  */
 export function encodeCompactIdToString(idArg: number | string, prefix = "") {
```

Comment:
Optional: Since it's one-way (there's no decode function), I'm wondering about a different name.  Like maybe `convertCompactIdToShortString` or `compressCompactId` or something like that?

Reply:
I like the idea. It's used across packages though - datastores, container-runtime so it's a non-trivial change.